functions.php

```diff
@@ -9,7 +15,11 @@ function printTiles($waves) {
 }
 }
 
-
+/*
+ * With each row being parsed out into its own array by the above loop, this function
+ * will put each of those array values into html format to be echoed in the
+ * index file.
+ */
 function displayWave($waveName, $wavePleasant, $waveGenuine, $wavePivot){
```

Comment:
I think the latest psr2 is to have the curly brackets for a function to start on a new line though I think that you owuld get away with it a mike didnt know that they had changed it